src/linear_algebra/free_module.lean

```diff
@@ -335,3 +335,14 @@ lemma submodule.exists_is_basis_of_le_span
 submodule.exists_is_basis_of_le le (is_basis_span hb)
 
 end principal_ideal_domain
+
+lemma algebra_map.injective.linear_independent {R S M ι : Type*} [comm_semiring R] [semiring S]
```

Comment:
This looks to me like a better name might be `linear_independent.restrict_scalars`

Reply:
Thanks: name updated!